docs/README.md

```diff
@@ -19,13 +15,39 @@ This Splunk distribution comes with the following defaults:
 - OTLP over HTTP exporter configured to send spans to a locally running [Splunk OpenTelemetry
   Collector](https://github.com/signalfx/splunk-otel-collector)
 
-Install the distribution:
+### Install
+
+Download and extract the appropriate binaries from
+[the latest release](https://github.com/signalfx/splunk-otel-dotnet/releases/latest).
 
-TODO
+> The path where you put the binaries is referenced as `$INSTALL_DIR`
 
 ### Basic configuration
 
-TODO
+When running your application, make sure to:
+
+1. Set the [resources](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.3.1-beta.1/docs/config.md#resources).
```

Comment:
I think that you are missing info about our plugins. I know that both are doing nothing, but it should be documented.

Reply:
How do you want to go about it? "Here are some plugins that do nothing"? 🤔